apps/expo-go/ios/Exponent.xcodeproj/project.pbxproj

```diff
@@ -1537,7 +1537,6 @@
 				"${PODS_CONFIGURATION_BUILD_DIR}/GoogleUtilities/GoogleUtilities_Privacy.bundle",
 				"${PODS_CONFIGURATION_BUILD_DIR}/PromisesObjC/FBLPromises_Privacy.bundle",
 				"${PODS_CONFIGURATION_BUILD_DIR}/PromisesSwift/Promises_Privacy.bundle",
-				"${PODS_CONFIGURATION_BUILD_DIR}/RCT-Folly/RCT-Folly_privacy.bundle",
```

Comment:
this could be unrelated change. please try to run the command to ensure cocoapods doesn't have any outdated cache
```
rm -rf apps/bare-expo/ios/Pods apps/bare-expo/ios/build apps/expo-go/ios/Pods apps/expo-go/ios/build && et pods -f
```